### ISSUE TITLE
Salir de los modos basico y avanzado

### ISSUE THREAD

**Issue by kibojoe** [dmaciasblog/formatusb#1]
muy buen el script! quisiera saber como hago para salir de los modos basico y avanzado. por ejemplo:

http://www.auplod.com/u/apodul9dc5c.png

como salir y volver al menu principal? creo que es importante esa función y no se como añadir esto!
saludos...

**Comment by dmaciasblog** [comment]
se podría añadir eso que quieres para el modo básico en la función comprobar, habría que añadir otro if que cuando lea $disp salga a menú_principal con una tecla en especial, por ejemplo Q (quit) y en la avanzada sería hacer lo mismo pero sobre la misma función avanzada, si quieres luego a la noche hago una prueba.
un saludo

**Comment by kibojoe** [comment]
@dmaciasblog 
gracias y voy a intentar aqui.
saludos...

**Comment by kibojoe** [comment]
@dmaciasblog 
que tal? estoy adaptando su script para el Kibojoe Linux (http://kibojoe.org/) y no trabaja la opicion salir de los modos basico y avanzado. podria ayudar en esto? mi adaptacion esta en

https://github.com/kibojoe/scripts-kibojoe/blob/master/kibojoe/bfusb

el image de **Backup Format USB Tool**

http://www.auplod.com/u/pulado9ddda.png

obs.: falta todavia infomationes en la opcion "i Information"...
saludos

**Comment by dmaciasblog** [comment]
claro, sería un placer hacer la modificación, esta tarde que no trabajo me pongo a ello
un saludo

**Comment by dmaciasblog** [comment]
@kibojoe ya he añadido el poder salir de los modos basico y avanzado, cualquier cosa que necesites me dices.
Con tu permiso voy a dar un vistazo a tu repo

Un saludo

**Comment by kibojoe** [comment]
gracias @dmaciasblog 
si puede mirar el repo...
saludos
